packages/rum-core/src/domain/assembly.spec.ts

```diff
@@ -516,7 +516,8 @@ describe('rum assembly', () => {
     })
 
     it('should detect synthetics sessions from global', () => {
-      ;(window as BrowserWindow)._DATADOG_SYNTHETICS_BROWSER = true
+      ;(window as BrowserWindow)._DATADOG_SYNTHETICS_PUBLIC_ID = 'foo'
+      ;(window as BrowserWindow)._DATADOG_SYNTHETICS_RESULT_ID = 'bar'
```

Comment:
I would create a utility function `setSyntheticsGlobals('foo', 'bar')` or similar to avoid repetition and make the test more expressive.
I would do the same and delete them with a `deleteSyntheticsGlobals()` utility function.